uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_absence-tab-page.scss

```diff
@@ -1,5 +1,19 @@
 .absence-tab-page {
 
+  /**
+   * This is a temporary fix.
+   * It provides a consistency to paddings and at the same time
+   * preserves the same maximum width for larger screens.
+   * @TODO this fix should be removed once all other
+   * screens on admin have consistent layout
+  */
+  >.container-fluid {
+    width: 99%;
+    padding-right: calc(8px + 1%);
```

Comment:
I can see that this is working but shouldn't width be something like `width: calc(99% - 8px)` ?

Reply:
@reneolivo I updated the PR description and it should now answer your question. I now also use viewport width instead of % since it give even higher precision.